packages/validate/src/lib/validator/utils.js

```diff
@@ -27,7 +27,7 @@ export const groupIsDisabled = fields => fields.reduce((acc, field) => {
 export const hasValue = input => (input.value !== undefined && input.value !== null && input.value.length > 0);
 
 export const groupValueReducer = (acc, input) => {
-    if (!isCheckable(input) && !isHidden(input) && hasValue(input)) acc = input.value;
+    if (!isCheckable(input) && !isHidden(input) && hasValue(input)) acc = input.value.trim();
```

Comment:
Would there be a case for applying this to the checkable inputs too?  They wouldn't be user input, but does the uncheckable inputs include a select box, which is in the same boat.

Reply:
I've added it to the checkable inputs too. I can't think of likely use-case for radio or checkbox needing it but that doesn't mean it couldn't happen, and it's a low risk addition.